### DISCUSSION TITLE
Release a version with ChatVertexAI enabled

### DISCUSSION THREAD

**Discussion by dvmorris** [langflow-ai/langflow#956] [Q&A]
ChatVertexAI is working on the dev branch, but it's not included in any of the recent releases of langflow. Can you release a version that has this included? Here is the original PR: https://github.com/logspace-ai/langflow/pull/724

**Comment by carlosrcoelho** [comment]
Hello!

Sorry for the delayed response.

Please check version v1.1.1, and if there's any problem, kindly open a new issue.
